crates/extensions/tedge_mqtt_bridge/src/lib.rs

```diff
@@ -336,10 +385,13 @@ async fn half_bridge(
             }
         };
         debug!("Received notification ({name}) {notification:?}");
+        debug!("Bridge {name} connection: received={received} forwarded={forwarded} published={published} waiting={} acknowledged={acknowledged} finalized={finalized} ignored={ignored}",
```

Comment:
This is a really nice way to observe what's going on without resorting to logging every event and trying to manually reconstruct what's going on.